scripts/ci/ci-run-sqllogic-tests.sh

```diff
@@ -17,4 +17,4 @@ fi
 echo "Run suites using argument: $RUN_DIR"
 
 echo "Starting databend-sqllogic tests"
-target/${BUILD_PROFILE}/databend-sqllogictests --handlers ${TEST_HANDLERS} ${RUN_DIR} --enable_sandbox --parallel 8 --debug
+target/${BUILD_PROFILE}/databend-sqllogictests --handlers ${TEST_HANDLERS} ${RUN_DIR} --skip_dir management --enable_sandbox --parallel 8 --debug
```

Comment:
Why skip management test?

Reply:
Tests in management dir need the server run on management mode, it does not affect the ci run cause our ci has a separate action(test_sqllogic_management_mode_linux) to run this test.
It just affect do `make sqllogic-test` on local machine.